docs/guides/writing_stubs.rst

```diff
@@ -113,18 +118,6 @@ Stub Content
 This section documents best practices on what elements to include or
 leave out of stub files.
 
-Modules excluded fom stubs
```

Comment:
I moved this down cuz it felt weird for the "stub content" section to start with "what you shouldn't include"